src/translations/en/filterExpertEn.ts

```diff
@@ -188,4 +188,6 @@ export const filterExpertEn = {
     YupNotTypeDefault: 'Field value format is incorrect',
     changeOperatorMessage:
         'The operator will be changed and will be applied to all the rules already created in the group.',
+    shortCircuitLowCurrentLimit: 'Low short-circuit current limit',
+    shortCircuitHighCurrentLimit: 'High short-circuit current limit',
```

Comment:
Not important but why do you reorganize the order of the words for the key ? (and for the enums everywhere)